pysmt/smtlib/parser/parser.py

```diff
@@ -449,6 +449,8 @@ def fix_real(op, *args):
 
         # Command tokens
         self.commands = {smtcmd.ASSERT : self._cmd_assert,
```

Comment:
Having them alphabetically is nice, but I would have a slight preference to move all the OMT specific commands at the end following a comment like:

```
# OMT Extension (http://optimathsat.disi.unitn.it/pages/smt2reference.html)
```